sw/tools/airframe_editor/xml_airframe.py

```diff
@@ -76,13 +92,18 @@ def reorganize_airframe_xml(airframe_xml):
     find_and_add_sections_with_name(airframe_xml, airframe, "XSENS")
 
     find_or_add_group(airframe_xml, airframe, "GAINS")
+
     # Fixedwing
-    find_and_add_sections_with_name(airframe_xml, airframe, "HORIZONTAL CONTROL")
+    find_and_add_sections_with_name(airframe_xml, airframe,
```

Comment:
I understand and I had the same point of view and researched where the 80 chars limit came from. Apparently, it comes from visualizing 'side by side' several files in editor like emacs. But in many other IDEs, 100 or 120 is fine. Maybe 80 should be condidered as a very soft rule that can be broken if readability is better.